src/main/java/org/jenkinsci/test/acceptance/docker/DockerContainerHolder.java

```diff
@@ -36,7 +38,7 @@
      */
     @Inject(optional = true)
     @Named("dockerPortOffset")
-    private int portOffset = 0;
+    private Integer portOffset;
```

Comment:
This way the semantic is clearer: we can use no port offset (null, which is default vale), or we can set an offset of 0, which means exact mapping. 

I leverage this to implement the `matchHostPorts` property in the docker fixture.